lib/modules/list-widgets/index.js

```diff
@@ -111,7 +81,7 @@ module.exports = {
         },
         {
           label: 'Numbers',
-          value: 'uk-list uk-list-numbers'
+          value: 'ordered-list'
```

Comment:
Dit maakt de backwards-compatibility stuk, toch? Want als er nu gestyled is op `.uk-list` of `.uk-list-numbers` dan ben je die styling kwijt (omdat je in feite je selector nu verandert).